src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -46,18 +52,26 @@ interface ChartsHolderProps {
   chartId: string;
 }
 
-const ChartsHolder = ({ region, chartId }: ChartsHolderProps) => {
+const ChartsHolder = React.memo(({ region, chartId }: ChartsHolderProps) => {
   const projections = useProjectionsFromRegion(region);
 
   const locationSummary = useLocationSummariesForFips(region.fipsCode);
 
-  const metricRefs = {
-    [Metric.CASE_DENSITY]: useRef<HTMLDivElement>(null),
-    [Metric.CASE_GROWTH_RATE]: useRef<HTMLDivElement>(null),
-    [Metric.POSITIVE_TESTS]: useRef<HTMLDivElement>(null),
-    [Metric.HOSPITAL_USAGE]: useRef<HTMLDivElement>(null),
-    [Metric.VACCINATIONS]: useRef<HTMLDivElement>(null),
-  };
+  const caseDensityRef = useRef<HTMLDivElement>(null);
+  const caseGrowthRateRef = useRef<HTMLDivElement>(null);
+  const positiveTestsRef = useRef<HTMLDivElement>(null);
+  const hospitalUsageRef = useRef<HTMLDivElement>(null);
+  const vaccinationsRef = useRef<HTMLDivElement>(null);
+  const metricRefs = useMemo(
+    () => ({
+      [Metric.CASE_DENSITY]: caseDensityRef,
+      [Metric.CASE_GROWTH_RATE]: caseGrowthRateRef,
+      [Metric.POSITIVE_TESTS]: positiveTestsRef,
+      [Metric.HOSPITAL_USAGE]: hospitalUsageRef,
+      [Metric.VACCINATIONS]: vaccinationsRef,
+    }),
+    [],
+  );
```

Comment:
This is a bit annoying / ugly, but this is the only way I could figure out how to memoize `metricRefs` so that I could memoize the charts, etc.